EIPS/eip-is-contract.md

```diff
@@ -0,0 +1,106 @@
+---
+title: IS_CONTRACT instruction
+description: Introduce IS_CONTRACT instruction
+author: Andrei Maiboroda (@gumb0), Piotr Dobaczewski (@pdobacz), Danno Ferrin (@shemnon)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians in the EIPs category using the template: https://ethereum-magicians.org/c/eips/5